documentation/modules/ova-prerequisites.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: REFERENCE
+[id="ova-prerequisites_{context}"]
+= Open Virtual Application (OVA) prerequisites
+
+The following prerequisites apply to Open Virtual Application (OVA) file migrations:
+
+* All OVA files are created by or compatible with VMware vSphere.
+* The OVA files are in a shared NFS share that has the following security settings: `rw`.
+* Each OVA file is in a separate folder in the NFS share -  sub-folders are not checked.
+* The files are in one of the following formats:
+
+** An OVA file that holds all the VM information
+** A folder containing the extracted OVA with the `.ovf`, `.vmdk` and `.mf` files available
```

Comment:
the specification says `zero or one OVF manifest with extension .mf` and also zero or more vmdk files - I'd try to avoid listing the content of the OVF here, if we say that the OVA is supposed to be generated by vSphere then it will have the needed resources